doc/man1/ciphers.pod

```diff
@@ -346,7 +346,7 @@ Cipher suites using SHA1.
 
 =item B<SHA256>, B<SHA384>
 
-Ciphersuites using SHA256 or SHA384.
+Cipher suites using SHA256 or SHA384.
```

Comment:
I've changed to _ciphersuite_, although that doesn't appear in the dictionaries I've checked (OED and Webster's).  This isn't an issue in code, but shouldn't the documentation use dictionary words where possible?